src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -182,6 +189,26 @@ def validate_step(self, step: str) -> bool:
                 return run_new_state is not None and run_new_state.result_state == RunResultState.SUCCESS
         return False
 
+    def _relay_logs(self, workflow, run_id):
+        for log in self._fetch_logs(workflow, run_id):
+            task_logger = logging.getLogger(log.component)
+            # we're relaying almost raw log events, so we need to call the logger directly
```

Comment:
Could you elaborate this? Why not use `task_logger.log`?

Reply:
It doesn't have ability to set level

Reply:
it does, right? https://docs.python.org/3/library/logging.html#logging.Logger.log

Reply:
oh, okay